mesheryctl/internal/cli/root/pattern/apply.go

```diff
@@ -55,6 +40,7 @@ var applyCmd = &cobra.Command{
 		}
 
 		res, err := client.Do(req)
+		fmt.Println(res.StatusCode)
```

Comment:
Might want to remove this print statement :)

Reply:
Good catch 👍